src/sinks/websocket/sink.rs

```diff
@@ -242,6 +242,7 @@ impl WebSocketSink {
         match self.encoder.serializer() {
             RawMessage(_) | Avro(_) | Native(_) | Protobuf(_) => true,
             Csv(_) | Logfmt(_) | Gelf(_) | Json(_) | Text(_) | NativeJson(_) => false,
+            vector_lib::codecs::encoding::Serializer::Syslog(_) => todo!(),
```

Comment:
@cahartma why is this here?

Reply:
Build fixes.  Wanted to discuss.   When I clone and simply `cargo check`, there are several issues needing resolved.

Reply:
Can we merge these unrelated necessities in a different(s) PRs?

Reply:
Yep good idea.   I'll submit another PR and attempt to clean up our vendoring and dependencies.

Reply:
https://github.com/ViaQ/vector/pull/198